server/views/pages/createAlertType/confirmation.njk

```diff
@@ -1,19 +1,77 @@
 {% extends "../../partials/layout.njk" %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
 
-{% set pageTitle = applicationName + " - Home" %}
+{% set pageTitle = applicationName + " - Create alert type confirm details" %}
 {% set mainClasses = "app-container govuk-body" %}
-
+{% block beforeContent %}
+    {{ govukBackLink({
+        text: "Back",
```

Comment:
Really minor and may well be deliberate. "Back" is the default text and you can only supply the href if you prefer

Reply:
I do like to be explicit...